CONTRIBUTING.md

```diff
@@ -85,7 +85,7 @@ because <additional rationale>.
 Add new Localization.lang("KEY") to Java file.
 Tests fail. In the test output a snippet is generated which must be added to the English translation file. There is also a snippet generated for the non-English files, but this is irrelevant.
 Add snippet to English translation file located at `src/main/resources/l10n/JabRef_en.properties`
-With `gradlew generateMissingTranslationKeys` the "KEY" is added to the other translation files as well.
+With `gradlew update` the "KEY" is added to the other translation files as well.
```

Comment:
Hm, I don't like the current task name. It should say what it does, like `updateLocalization`. Please also change the other Gradle tasks accordingly.